Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -19,6 +19,37 @@ final class DemoMerchantAPI {
 
     // MARK: Public Methods
     
+    func getSetupToken(customerID: String? = nil, selectedMerchantIntegration: MerchantIntegration) async throws -> SetUpTokenResponse? {
+        let request = SetUpTokenRequest(customerID: customerID)
+        let urlRequest = try createSetupTokenUrlRequest(
+            setupTokenRequest: request, environment: DemoSettings.environment, selectedMerchantIntegration: selectedMerchantIntegration
+        )
+        
+        do {
+            let data = try await data(for: urlRequest)
+            return try parse(from: data)
+        } catch {
+            print("error with the create setup token request: \(error.localizedDescription)")
+            return nil
+        }
+    }
+    
+    func getPaymentToken(setupToken: String, selectedMerchantIntegration: MerchantIntegration) async throws -> PaymentTokenResponse? {
+        // make responseType
+        let request = PaymentTokenRequest(setupToken: setupToken)
+        let urlRequest = try createPaymentTokenUrlRequest(
+            paymentTokenRequest: request, environment: DemoSettings.environment, selectedMerchantIntegration: selectedMerchantIntegration
+        )
+        
+        do {
+            let data = try await data(for: urlRequest)
+            return try parse(from: data)
+        } catch {
+            print("error with the create setup token request: \(error.localizedDescription)")
+            return nil
```

Comment:
Same question here should this be `throw error` or do we want to return nil?

Reply:
03ecf9ae0cba712d182793350a8c69849136aac5